source/1_engine/rex_engine/src/memory/memory_tracking.cpp

```diff
@@ -17,27 +23,79 @@ namespace rex
 
   rsl::array<MemoryTag, g_max_allowed_mem_tags>& thread_local_memory_tag_stack()
   {
-    thread_local static rsl::array<MemoryTag, g_max_allowed_mem_tags> stack = {MemoryTag::Global};
+    thread_local static rsl::array<MemoryTag, g_max_allowed_mem_tags> stack = { MemoryTag::Global };
     return stack;
   }
 
+  class AllocationCallStack
+  {
+  public:
+    AllocationCallStack(CallStack callstack, card64 size)
```

Comment:
We want to copy by value here, shouldn't we at least move them into the datamember then?

Reply:
I can move it, making it future proof, but moving has no effect here